package.json

```diff
@@ -7,6 +7,7 @@
     "build:ci": "cross-env REACT_APP_CI=1 react-scripts build",
     "coverage": "cross-env NODE_ICU_DATA=node_modules/full-icu react-scripts test --coverage --watchAll=false",
     "eject": "react-scripts eject",
+    "validate-branch-name": "chmod 755 ./scripts/validate-branch-name.sh && bash ./scripts/validate-branch-name.sh",
```

Comment:
I am not 100% sure if this scenario is even available, but I assume that our boilerplate is open source so there is some chance that not unix users will have problem with bash scripts execution

Reply:
Unfortunately I've no experience in scripting on MacOS or Windows - I'm not sure how to do that and I won't be able to test it.

Reply:
I suggest leaving this like it is but adding extra info in README that it won't work in windows environment